### ISSUE TITLE
Pandas Incompatible With Numpy

### ISSUE THREAD

**Issue by pjockey** [rdefeo/stonks#11]
Installed using the script and installed ok with no errors. I added"stonks" into the nhl-led-scoreboard/config/config.json file, saved and then tried to start the nhl board and received the following errors. Installed version of numpy is 1.16.6.
```
./startup.sh
Traceback (most recent call last):
  File "src/main.py", line 5, in <module>
    from renderer.main import MainRenderer
  File "/home/pi/nhl-led-scoreboard/src/renderer/main.py", line 5, in <module>
    from boards.boards import Boards
  File "/home/pi/nhl-led-scoreboard/src/boards/boards.py", line 20, in <module>
    from boards.stonks import Stonks
  File "/home/pi/nhl-led-scoreboard/src/boards/stonks.py", line 6, in <module>
    import yfinance as yf
  File "/usr/local/lib/python3.7/dist-packages/yfinance/__init__.py", line 23, in <module>
    from .ticker import Ticker
  File "/usr/local/lib/python3.7/dist-packages/yfinance/ticker.py", line 27, in <module>
    import pandas as _pd
  File "/usr/local/lib/python3.7/dist-packages/pandas/__init__.py", line 22, in <module>
    from pandas.compat.numpy import (
  File "/usr/local/lib/python3.7/dist-packages/pandas/compat/numpy/__init__.py", line 21, in <module>
    f"this version of pandas is incompatible with numpy < {_min_numpy_ver}\n"
ImportError: this version of pandas is incompatible with numpy < 1.16.5
your numpy version is 1.16.2.
Please upgrade numpy to >= 1.16.5 to use this pandas version
```

**Comment by rdefeo** [comment]
This was recently fixed (by adding the appropriate entry in the requirements.txt file) by the implementation of Issue #1, which included the pull request #10

Note: You mention that your installed version of numpy is 1.16.6, but the script is reporting 1.16.2, hence the error.

**Comment by pjockey** [comment]
Where does the script pull that info from? I've upgrade both pip and pip3 to the 1.16.6 using both sudo and no sudo
